src/documentation/templates/modules/content/icon-page.html

```diff
@@ -1,6 +1,10 @@
 <div class="docs-IconList clearfix">
```

Comment:
How is this icon page generated? We have a [JSON data file for icons](https://github.com/OfficeDev/office-ui-fabric-core/blob/master/src/data/icons.json) that will always be up-to-date with the latest icons in Fabric, and would be good to use here.